### PR TITLE
STADTPULS-691 wenn dies dann das story

### PR DESCRIPTION
**Pull request by ff6347**
First draft. See Jira issue for further infos. 
Also fixes a small issue in the sensor card src/common/sensor-cards.js The script did not display the symbol of the sensor right. it used the sensor.id instead of sensor.icon_id

### CODE REVIEW

**Review by vogelino — CHANGES_REQUESTED** [review]
Great work! I followed the tutorial and it was easy to follow along and it worked well.

I noticed a few things however:
- The images don't load on the deployed versions. Their `src` attributes should include a forward slash for them to work
- There is no link to your story in the root page. In `/src/index.html`, copy a `a.stories-list-item` element and add your story info
- There is still an unwritten section. Search for TBD in your file.
- I would find it better if you author picture was made like all the others. See how to create one in the README file
- You still need to add a custom social-picture for you story. See the README on how to do so and see how it's done in the wrangelkiez story (`src/wrangelkiez/images/social-image.jpg`)

**PS:** I checked neither the grammar nor the writing style as I think that @Esshahn is better suited to take care of this part.

File: src/wenn-dies-dann-das/index.html [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+            <h2 id="sensor-und-token">Sensor und Token</h2>
+            <p>
+              Nachdem wir all unsere Accounts erstellt haben, legen wir zuallererst in
+              unserem Stadtpuls.com Account einen neuen Sensor an. Als Kategorie wählen wir
+              “Sonstige” und als Integration “HTTP”. Alle anderen Werte und Namen seien dem
+              Leser überlassen.
+            </p>
+
+            <figure>
+              <img src="images/new-sensor.png" alt="Interface zum erstellen eines neuen Sensors" />
```
[the remark this answers is not in the document]
Comment (ff6347) [reply src/wenn-dies-dann-das/index.html:251]: Wired thing. It worked for me after reloading the page once more. What do you mean by forward slash? ./images/....png right?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,9 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode/settings.json
+node_modules
+src/wenn-dies-dann-das/draft.html
+src/wenn-dies-dann-das/pandoc.sh
```
[the remark this answers is not in the document]
Comment (vogelino) [reply .gitignore:5]: You can use your global or personal configuration for this kind of use cases. The .gitignore has also a documentation dimension to it, so I believe it should remain free from personal configuration. But I'm not too religious about it. Feel free to ignore.

File: src/wenn-dies-dann-das/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (vogelino) [reply src/wenn-dies-dann-das/index.html:251]: Yes but without the dot. For example: /img/image.png

[the remark this answers is not in the document]
Comment (ff6347) [reply src/wenn-dies-dann-das/index.html:251]: But then I have to give the full path. /wenn-dies-dann-dass/images/....png The relative paths should be fine.

[the remark this answers is not in the document]
Comment (vogelino) [reply src/wenn-dies-dann-das/index.html:251]: Whatever works. Personally it doesn't when I open the vercel preview so something is wrong 😑

File: .gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (ff6347) [reply .gitignore:5]: gone

File: src/wenn-dies-dann-das/index.html [hunk 3]
```diff
@@ -0,0 +1,690 @@
...
+              Markdown, HTML, TeX und sogar SQL Datenbanken anlegen können um unsere
+              Notebooks mit Informationen und Daten anzureichern. Vergleichbare Plattformen
+              wären hier Google Colab, deepnote.com, Juypter Notebook oder RStudio et. al.
+            </p>
+            <h3 id="was-ist-mapbox.com">Was ist Mapbox.com</h3>
+            <p>
+              Mapbox ist auch eine kommerzielle Plattformen die im Bereich interaktive
+              Karten den Ton angibt.
+            </p>
+            <p>TBD</p>
```
[the remark this answers is not in the document]
Comment (ff6347) [reply src/wenn-dies-dann-das/index.html:241]: Done

**Comment by ff6347** [comment]
image path is fixed

**Comment by vogelino** [comment]
Great! Thx for taking my feedback into consideration!

PS: Not that there is still no thumbnail on the overview page of the stories page. Don't forget to add it! :)

**Review by vogelino — APPROVED** [review]

**Comment by Esshahn** [comment]
can you add your correct twitter?

**Comment by ff6347** [comment]
> can you add your correct twitter?

Ähm. I thought I added it. it is https://twitter.com/fmoronzirfas

But I saw that the share links are wrong.

https://github.com/technologiestiftung/stadtpuls-story-template/blob/dacb8c62be8afe81e0484ed1abfab4143b36f318/src/wenn-dies-dann-das/index.html#L516-L542

**Comment by Esshahn** [comment]
looks great!

**Comment by ff6347** [comment]
It is so nice to sit on the sofa while enjoying a coffee and to see how you fix the mess I made. Thanks a lot guys. I'd love to have you listed as co authors.

**Comment by Esshahn** [comment]
that's what friends are for!

**Review by Esshahn — APPROVED** [review]

**Pull request merged**